cpp/include/Ice/ServerAuthenticationOptions.h

```diff
@@ -0,0 +1,415 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#ifndef ICE_SSL_SERVER_AUTHENTICATION_OPTIONS_H
+#define ICE_SSL_SERVER_AUTHENTICATION_OPTIONS_H
+
+#include "SSLConfig.h"
+#include "SSLConnectionInfo.h"
+
+#include <functional>
+
+namespace Ice::SSL
+{
+    /**
+     * The SSL configuration properties for server connections.
+     */
+    struct ServerAuthenticationOptions
+    {
+#if defined(ICE_USE_SCHANNEL)
```

Comment:
Why is it ICE_USE_SCHANNEL here and WIN32 in the ClientAuthenticationOptions.h?

Reply:
I added these defines later in SSLConfig.h, and forgot to upgrade ClientAuthenticationOptions